middlewares/inputValidator.js

```diff
@@ -0,0 +1,37 @@
+
+
+/** * Class validating user inputs */
+export default class userValidator {
+  /**
+  * validates email
+  * @callback next
+  * @param {object} req - Request object
+  * @param {Object} res - Response object
+  * @param {next} next - Next route handler
+  * @returns {Object} failure message
+  */
+  static validateEmail(req, res, next) {
+    req.sanitizeBody('user.email').trim();
+    req.checkBody('user.email', 'Invalid email address').isEmail();
+    const errors = req.validationErrors();
+    if (errors) {
+      return res.status(422).json({ errors: { body: [errors[0].msg] } });
+    }
+    next();
+  }
+
+  /**
+  * validates password
+  * @callback next
+  * @param {Object} req - Request object
+  * @param {Object} res - Response object
+  * @param {next} next - Next route handler
+  * @returns {Object} failure message
+  */
+  static validatePassword(req, res, next) {
```

Comment:
You can use the validate password method in the validate helper class to validate the password here, it properly checks for the right password formats.

Reply:
I have implemented the change requested